integreat_cms/cms/templates/events/event_list.html

```diff
@@ -40,53 +40,65 @@ <h1 class="heading">{% trans 'Events' %}</h1>
     </div>
     {% endwith %}
     <div class="table-listing">
-        <table id="event-list" class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
-            <thead>
-            <tr class="border-b border-solid border-gray-200">
-                <th class="text-sm text-left uppercase py-3 pl-4 pr-2">{% trans 'Title in' %} {{ language.translated_name }}</th>
-                {% get_current_language as LANGUAGE_CODE %}
-                {% get_language LANGUAGE_CODE as backend_language %}
-                {% if backend_language and backend_language != language %}
-                    <th class="text-sm text-left uppercase py-3 pr-2">{% trans 'Title in' %} {{ backend_language.translated_name }}</th>
-                {% endif %}
-                <th class="text-sm text-left uppercase py-3">
-                    <div class="lang-grid flags whitespace-nowrap">
-                        {% spaceless %}
-                            {% for lang in languages %}
-                                <a href="{% url 'events' region_slug=region.slug language_slug=lang.slug %}">
-                                    <span class="fp fp-rounded fp-{{ lang.primary_country_code }}" title="{{ lang.translated_name }}"></span>
-                                </a>
-                            {% endfor %}
-                        {% endspaceless %}
-                    </div>
-                </th>
-                <th class="text-sm text-left uppercase py-3 pr-2">{% trans 'Status' %}</th>
-                <th class="text-sm text-left uppercase py-3 pr-2">{% trans 'Event location' %}</th>
-                <th class="text-sm text-left uppercase py-3 pr-2">{% trans 'Start' %}</th>
-                <th class="text-sm text-left uppercase py-3 pr-2">{% trans 'End' %}</th>
-                <th class="text-sm text-left uppercase py-3 pr-2">{% trans 'Recurrence' %}</th>
-                <th class="text-sm text-right uppercase py-3 pr-4 min">{% trans 'Options' %}</th>
-            </tr>
-            </thead>
-            <tbody>
-            {% if events %}
-                {% for event in events %}
+        <form id="bulk-action-form" method="post">
+            <table id="event-list" class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
+                <thead>
+                <tr class="border-b border-solid border-gray-200">
+                    <th class="py-3 pl-2 inline-block text-gray-800 lang-grid">
+                        <input type="checkbox" id="bulk-select-all">
+                    </th>      
+                    <th class="text-sm text-left uppercase py-3 pl-4 pr-2">{% trans 'Title in' %} {{ language.translated_name }}</th>
```

Comment:
```suggestion
                    <th class="text-sm text-left uppercase py-3 pl-2 pr-2">{% trans 'Title in' %} {{ language.translated_name }}</th>
```
Since this column isn't the first one in the table anymore, `pl-4` is a bit too much here I think...